recipe/0001-use-protobuf-dlls.patch

```diff
@@ -0,0 +1,27 @@
+From f5397398a0d697bcefb10d7be589fc215f3abc6a Mon Sep 17 00:00:00 2001
+From: Ista Zahn <izahn@hbs.edu>
+Date: Mon, 19 Sep 2022 06:32:25 -0400
+Subject: [PATCH] use protobuf dlls
+
+---
+ src/core/CMakeLists.txt | 4 ++++
+ 1 file changed, 4 insertions(+)
+
+diff --git a/src/core/CMakeLists.txt b/src/core/CMakeLists.txt
+index 7537413..8eed55d 100644
+--- a/src/core/CMakeLists.txt
++++ b/src/core/CMakeLists.txt
+@@ -1813,6 +1813,10 @@ if (NOT MSVC)
+   set_source_files_properties(${VECTOR_TILE_PROTO_SRCS} PROPERTIES COMPILE_FLAGS -w)
+ endif()
+ 
++if (MSVC)
++  set_source_files_properties(${VECTOR_TILE_PROTO_SRCS} PROTOBUF_USE_DLLS)
```

Comment:
Should be:
```
set_source_files_properties(${VECTOR_TILE_PROTO_SRCS} vectortile/qgsvectortilewriter.cpp PROPERTIES COMPILE_DEFINITIONS PROTOBUF_USE_DLLS)
```

Reply:
I see you fixed this in https://github.com/conda-forge/qgis-feedstock/pull/253/commits/82a74382cbdd911eec68f7a359ab4c7f5f4ae006 thank you!

Reply:
Sorry - I think I gave you the wrong information! I'll try and get this upstreamed (again).